### ISSUE TITLE
Resume pre-training using "resume_from_checkpoint" in pytorch lightning

### ISSUE THREAD

**Issue by haoyusimon** [zdou0830/METER#37]
Hi, thank you for sharing the code! I have a question regarding how to resume the pre-training. My pre-training stopped halfway and I want to resume the pre-training from the checkpoint saved. Have you tried to resume pre-training from a checkpoint when doing experiments? And may I know what is the method you used? I have tried to set the directory "resume_from=" in the config.py file (which sets the "resume_from_checkpoint" parameter in PyTorch lightning trainer) to resume training, but I found that the pertaining loss started to increase instead of decrease after one epoch of pre-training. Does this issue have something to do with the use of DDP? Thank you!

**Comment by zdou0830** [comment]
Hi, I tried to avoid resuming during training but I think using "resume_from" should be good and you can test the checkpoints on downstream tasks to see if there are indeed issues.

**Comment by haoyusimon** [comment]
I see... Thank you so much for your reply!
